dev/TabView/TabView.idl

```diff
@@ -116,18 +109,33 @@ unsealed runtimeclass TabViewItem : Windows.UI.Xaml.Controls.ListViewItem
 
     Windows.UI.Xaml.DataTemplate HeaderTemplate{ get; set; };
 
-    Windows.UI.Xaml.Controls.IconElement Icon{ get; set; };
+    [MUX_PROPERTY_CHANGED_CALLBACK(TRUE)]
+    IconSource IconSource{ get; set; };
 
     [MUX_DEFAULT_VALUE("true")]
     [MUX_PROPERTY_CHANGED_CALLBACK(TRUE)]
     Boolean IsCloseable{ get; set; };
 
+    TabViewItemTemplateSettings TabViewTemplateSettings{ get; };
+
     event Windows.Foundation.TypedEventHandler<TabViewItem, TabViewTabClosingEventArgs> TabClosing;
 
     static Windows.UI.Xaml.DependencyProperty HeaderProperty{ get; };
     static Windows.UI.Xaml.DependencyProperty HeaderTemplateProperty{ get; };
-    static Windows.UI.Xaml.DependencyProperty IconProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty IconSourceProperty{ get; };
     static Windows.UI.Xaml.DependencyProperty IsCloseableProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty TabViewTemplateSettingsProperty{ get; };
+}
+
+[WUXC_VERSION_PREVIEW]
+[webhosthidden]
+unsealed runtimeclass TabViewItemTemplateSettings : Windows.UI.Xaml.DependencyObject
```

Comment:
Just to make sure, this was also put through API review, right?  I found out the last time I added a template settings class that they also need to go through code review despite being ultimately just an implementation detail, so I wanted to make sure.

Reply:
Well... no, admittedly. I didn't know it needed to. I'll email Mike about it then.

Reply:
Glad I asked, then. :)  I didn't know these needed to go through API review either.